dotnet/src/Connectors/Connectors.Memory.Redis/RedisMemoryStore.cs

```diff
@@ -200,31 +232,9 @@ public async Task RemoveBatchAsync(string collectionName, IEnumerable<string> ke
 
     #region constants  ================================================================================
 
-    /// <summary>
-    /// Vector similarity index algorithm. The default value is "HNSW".
-    /// <see href="https://redis.io/docs/stack/search/reference/vectors/#create-a-vector-field"/>
-    /// </summary>
-    internal const Schema.VectorField.VectorAlgo VECTOR_INDEX_ALGORITHM = Schema.VectorField.VectorAlgo.HNSW;
-
-    /// <summary>
-    /// Vector type. Supported types are FLOAT32 and FLOAT64. The default value is "FLOAT32".
-    /// </summary>
-    internal const string VECTOR_TYPE = "FLOAT32";
-
-    /// <summary>
-    /// Supported distance metric, one of {L2, IP, COSINE}. The default value is "COSINE".
-    /// </summary>
-    internal const string VECTOR_DISTANCE_METRIC = "COSINE";
-
-    /// <summary>
-    /// Query dialect. To use a vector similarity query, specify DIALECT 2 or higher. The default value is "2".
-    /// <see href="https://redis.io/docs/stack/search/reference/vectors/#querying-vector-fields"/>
-    /// </summary>
-    internal const int QUERY_DIALECT = 2;
```

Comment:
I think we should keep these constants. It would be great to do the following:
1. Convert access modifiers from `internal` to `private`.
2. Change naming to follow .NET naming convention (e.g. `QUERY_DIALECT` => `QueryDialect`)
3. Use it in constructors as default parameters (e.g. `RedisMemoryStore(..., int queryDialect = QueryDialect`)

Same applies to all other constants in this class.